go/vt/wrangler/traffic_switcher.go

```diff
@@ -1759,3 +1819,43 @@ func reverseName(workflow string) string {
 	}
 	return workflow + reverse
 }
+
+func (ts *trafficSwitcher) addParticipatingTablesToKeyspace(ctx context.Context, keyspace, tableSpecs string) error {
+	var err error
+	var vschema *vschemapb.Keyspace
+	vschema, err = ts.wr.ts.GetVSchema(ctx, keyspace)
+	if err != nil {
+		return err
+	}
+	if vschema == nil {
+		return fmt.Errorf("no vschema found for keyspace %s", keyspace)
+	}
+	if strings.HasPrefix(tableSpecs, "{") {
```

Comment:
What does `{` stand for? What makes the string special when there's `{`?

Reply:
added comment in the code: essentially the user can specify a proper vschema for a sharded keyspace or just a list of tables. The vschema is specified as an inline json string.